CMakeLists.txt

```diff
@@ -619,6 +619,9 @@ endif()
 
 list(APPEND EXTRA_LIBRARIES ${CMAKE_DL_LIBS})
 
+if(ANDROID)
```

Comment:
How does this play with `find_library(ATOMIC atomic)` in the code block below if you wanted to compile using clang? Unfortunately I can't decipher whether it would overwrite `set(ATOMIC libatomic.a)` or not.